OpenRA.Game/Map/Map.cs

```diff
@@ -760,9 +802,8 @@ public void FixOpenAreas(Ruleset rules)
 
 		public byte GetTerrainIndex(CPos cell)
 		{
-			var uv = cell.ToMPos(this);
-			var custom = CustomTerrain[uv];
-			return custom != byte.MaxValue ? custom : cachedTileSet.Value.GetTerrainIndex(MapTiles.Value[uv]);
+			var custom = CustomTerrain[cell];
+			return custom != byte.MaxValue ? custom : cachedTileSet.Value.GetTerrainIndex(MapTiles.Value[cell]);
```

Comment:
I can't see where the conversion is done twice in the case of isometric maps. Could you point out an example of this in which it happens?